### DISCUSSION TITLE
Playtime: Save ignited  state of slots when Reaper project is closed.  (Feature Request)

### DISCUSSION THREAD

**Discussion by atencion** [helgoboss/helgobox#1048] [Ideas]
It would be great if the playback (or "ignited") state of slots is saved as part of the project. It used to be the case in Playtime 1, and would make sense to save it in Playtime 2 as well.  

Thank you!

**Comment by Akognito** [comment]
without this great inconvenience!!!

**Comment by atencion** [comment]
Yes, Playtime 1 used to work like this.
